components/org.wso2.carbon.identity.api.server.cors/org.wso2.carbon.identity.api.server.cors.v1/src/main/resources/configs.yaml

```diff
@@ -0,0 +1,232 @@
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /cors/origins/{cors-origin-id}/apps:
+    get:
+      tags:
+        - App associations
+      summary: Retrieve the Applications associated with a CORS origin by ID.
+      operationId: getAssociatedAppsByCORSOrigin
+      description: Retrieve the Applications associated with a CORS origin by ID.
+      parameters:
+        - name: cors-origin-id
+          in: path
+          description: ID of a CORS origin
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CORSOriginAppsGetResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: carbon.super
+components:
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://localhost:9443/oauth2/authorize'
+          tokenUrl: 'https://localhost:9443/oauth2/token'
+          scopes: {}
+  parameters:
+    typeQueryParam:
+      in: query
+      name: type
+      required: false
+      description: |
+        Type of authenticators. Can be either 'LOCAL' or 'REQUEST_PATH'
+      schema:
+        type: string
+  schemas:
+    Error:
+      type: object
+      properties:
+        code:
+          type: string
+          example: AAA-00000
+        message:
+          type: string
+          example: Some Error Message
+        description:
+          type: string
+          example: Some Error Description
+        traceId:
+          type: string
+          example: e0fbcfeb-3617-43c4-8dd0-7b7d38e13047
+    CORSOriginsGetResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/CORSOriginGetObject'
+    CORSOriginGetObject:
```

Comment:
shall we rename this to CORSOrigin?

Reply:
Then it clashes with the `CORSOrigin` in the osgi and one has to be used with the f.q. name.

Reply:
@emswbandara Currently, this is named `CORSOriginObject` due to the recent changes in the API? WDYT?
Any suggestions for names?